Creat tables and index

karam084 opened the pull request:
**Project requirements**

- Take a look at the following database diagram for a clinic:
  ![clinic_diagram](https://user-images.githubusercontent.com/77942746/192651203-d38ad847-7765-421b-b219-8932b1f7522e.png)
- Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Remember to add the FK indexes.

karam084 committed:
Creat tables and index

karam084 committed:
Update readme file

karam084 committed:
Add schema attached to readme

karam084 committed:
Update readme with links

Ismailco reviewed (commented):
Hi @karam084, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that should be created using peer programming, all students in the group should contribute to this PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

karam084 committed:
Update README.md

Removed my partner from the readme file and work solo as Microverse required from me

karam084 commented:
> Hi @karam084,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that should be created using peer programming, all students in the group should contribute to this PR
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._
